### ISSUE TITLE
Laravel Twilio Video Conference

### ISSUE THREAD

**Issue by Lambertn33** [twilio/twilio-video.js#1894]
I am trying to integrate twilio in my application 
here is my code:

<script src="//media.twiliocdn.com/sdk/js/video/v1/twilio-video.min.js"></script>
<script>
    Twilio.Video.createLocalTracks({
       audio: true,
       video: { width: 300 }
    }).then(function(localTracks) {
       return Twilio.Video.connect('{{ $accessToken }}', {
           name: '{{ $roomName }}',
           tracks: localTracks,
           video: { width: 300 }
       });
    }).then(function(room) {
       console.log('Successfully joined a Room: ', room.name);

       room.participants.forEach(participantConnected);

       var previewContainer = document.getElementById(room.localParticipant.sid);
       if (!previewContainer || !previewContainer.querySelector('video')) {
           participantConnected(room.localParticipant);
       }

       room.on('participantConnected', function(participant) {
           participantConnected(participant);
       });

       room.on('participantDisconnected', function(participant) {
           participantDisconnected(participant);
       });
    });
    
    function participantConnected(participant) {
        const div = document.createElement('div');
        div.id = participant.sid;
        div.setAttribute("style", "float: left; margin: 10px;");

        participant.tracks.forEach(function(track) {
            trackAdded(div, track)
        });

        participant.on('trackAdded', function(track) {
            trackAdded(div, track)
        });
        participant.on('trackRemoved', trackRemoved);

        document.getElementById('media-div').appendChild(div);
    }

    function participantDisconnected(participant) {
        participant.tracks.forEach(trackRemoved);
        document.getElementById(participant.sid).remove();
    }

    function trackAdded(div, track) {
        div.appendChild(track.attach());
        var video = div.getElementsByTagName("video")[0];
        if (video) {
            video.setAttribute("style", "max-width:300px;");
        }
    }

    function trackRemoved(track) {
        track.detach().forEach( function(element) { element.remove() });
    }
</script>

but i am getting this error:   WebSocket connection to 'wss://endpoint.twilio.com/' failed:

could anyone help please?

**Comment by PikaJoyce** [comment]
Hi @Lambertn33,

Thanks for opening this issue. A few questions:
1. Could you please ensure that you have the correct credentials set up in your application?
2. Could you try it out with the [quickstart](https://github.com/twilio/video-quickstart-js) to ensure that you have the proper set up? 
3. It's also difficult to decipher, but can you ensure that `'{{ $accessToken }}'` is the correct token value (if your credentials are providing a valid token)? 

Please refer to the [`connect` API documentation](https://sdk.twilio.com/js/video/releases/2.24.2/docs/module-twilio-video.html#.connect__anchor) as well.

Best,
Joyce

**Comment by Lambertn33** [comment]
@PikaJoyce everything is good but i can't even create a room

**Comment by PikaJoyce** [comment]
Hello @Lambertn33,

Have you tested this out on the Quickstart application as well? Are you getting the same error?

Also, are there any additional messages that follow your error message? There should be some text within the `<error message>` area.
```WebSocket connection to 'wss://endpoint.twilio.com/' failed: <error message>```

Best,
Joyce

**Comment by Lambertn33** [comment]
@PikaJoyce Here is the full error that i am getting
![Capture](https://user-images.githubusercontent.com/60555392/193658655-5acbf0d1-48a9-4781-a4bb-9f73bc4f2491.PNG)

**Comment by PikaJoyce** [comment]
Hi @Lambertn33 

Could you also please try out the [react app](https://github.com/twilio/twilio-video-app-react) and plug in your credentials to test?

Best,
Joyce

**Comment by HJTandel** [comment]
> Hello @Lambertn33,
> 
> Have you tested this out on the Quickstart application as well? Are you getting the same error?
> 
> Also, are there any additional messages that follow your error message? There should be some text within the `<error message>` area. `WebSocket connection to 'wss://endpoint.twilio.com/' failed: <error message>`
> 
> Best, Joyce

i am gettign sam error how can i solve it ??
